### PR TITLE
Typography, language and merge with master

### PR DESCRIPTION
**Pull request by jonmv**
@kkraune please review and merge.

### CODE REVIEW

File: documentation/tutorials/blog-search.md [hunk 1]
```diff
@@ -705,7 +697,7 @@ For now we will only do a very simple grouping query to get a list of unique
 values for `date` ordered by the number of documents they occur in and top 3 is
 shown:
 
-   $ curl -s "http://localhost:8080/search/?yql=select%20*%20from%20sources%20*%20where%20sddocname%20contains%20%22blog_post%22%20limit%200%20%7C%20all(group(date)%20max(3)%20order(-count())each(output(count())))%3B" | python -m json.tool
+    curl -s 'localhost:4080/search/?yql=select%20*%20from%20sources%20*%20where%20sddocname%20contains%20%22blog_post%22%20limit%200%20%7C%20all(group(date)%20max(3)%20order(-count())each(output(count())))%3B' | python -m json.tool
```
Comment (kkraune) [inline documentation/tutorials/blog-search.md]: 8080 is the correct port

**Comment by jonmv** [comment]
Didn't finish the whole document, but will try to do so the coming days.

**Comment by kkraune** [comment]
please set 8080 port, then merge. thanks for cleaning up!

**Comment by jonmv** [comment]
Just minor stuff! In enjoy the reading, myself :]

**Pull request merged**